### PR TITLE
SB-6387: Display tenant logo on public pages if configured

### PR DESCRIPTION
**Pull request by revathitarento**

### CODE REVIEW

File: src/app/client/src/app/app.component.spec.data.ts [hunk 1]
```diff
@@ -1135,6 +1135,18 @@ export const mockData = {
...
             }
         }
+    },
+    tenantResponse: {
+        'id': 'api.tenant.info',
+        'params': {resmsgid: 'c33a73c0-a0b6-11e8-8b56-a93ad7c7024d', msgid: null, status: 'successful'},
+        'responseCode': 'OK',
+        'result': {
+            'titleName': 'rj',
+            'logo': 'https://staging.open-sunbird.org/assets/images/sunbird_logo.png',
```
Comment (rajeevsathish) [inline src/app/client/src/app/app.component.spec.data.ts]: multiple key has same value why can' t it be a like
x:
Y:
z: "ABC"

File: src/app/client/src/app/app.component.spec.ts [hunk 2]
```diff
@@ -122,6 +122,20 @@ describe('AppComponent', () => {
     expect(document.title).toBe(mockData.tenantSuccess.result.titleName);
     expect(document.querySelector).toHaveBeenCalled();
   });
+  it('Should display the tenant logo if user is not logged in', () => {
+    const userService = TestBed.get(UserService);
```
Comment (rajeevsathish) [inline src/app/client/src/app/app.component.spec.ts]: why are you creating the const when you are not using it

File: src/app/client/src/styles.css [hunk 3]
```diff
@@ -1145,7 +1145,12 @@ i.large.icon.limitedPublishingLinkIcon {
   width: auto;
   height: auto;
 }
-
+.tenant-header-logo-before-login {
```
Comment (rajeevsathish) [inline src/app/client/src/styles.css]: is the Css approved by the UI team and the PM

File: src/app/client/src/app/app.component.spec.data.ts (hunk 1 above)
Comment (revathitarento) [reply src/app/client/src/app/app.component.spec.data.ts]: updated different values for mockdata

File: src/app/client/src/app/app.component.spec.ts (hunk 2 above)
Comment (revathitarento) [reply src/app/client/src/app/app.component.spec.ts]: removed the unused constant

File: src/app/client/src/styles.css (hunk 3 above)
Comment (revathitarento) [reply src/app/client/src/styles.css]: Css is updated as per the ui style guide, (Please check the attachment here - https://project-sunbird.atlassian.net/browse/SB-6387)

**Pull request merged**